src/core/ItemAtt.h

```diff
@@ -40,7 +40,7 @@ class CItemAtt : public CItem
 public:
   // The maximum supported attachment size
   // according to section 3.4 of formatV4 specification.
-  const static size_t MAX_SIZE = 4294967296; // 2^32
+  const static size_t MAX_SIZE = 4294967295U; // 2^32
```

Comment:
See https://en.cppreference.com/w/cpp/language/types and https://rules.sonarsource.com/c/RSPEC-854